docs/_extra/api-reference/hypothesis-v1.yaml

```diff
@@ -895,8 +897,12 @@ paths:
         - $ref: '#/components/parameters/GroupID'
         - $ref: '#/components/parameters/UserID'
       responses:
-        '204':
-          $ref: '#/components/responses/NoContent'
+        '200':
```

Comment:
I guess this is a significant change to the API but in practice I don't think clients will check for 204 in particular (vs any other 2XX code).